### PR TITLE
OCPBUGS#9250: Set CPU quota as false

### PR DESCRIPTION
**Pull request by xenolinux**
Version(s): 4.10+

Issue: https://issues.redhat.com/browse/OCPBUGS-9250

Link to docs preview: https://63734--docspreview.netlify.app/openshift-enterprise/latest/nodes/clusters/nodes-cluster-overcommit.html#nodes-cluster-overcommit-node-enforcing_nodes-cluster-overcommit

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by xenolinux** [comment]
@sunilcio Can you PTAL?

File: modules/nodes-cluster-overcommit-node-enforcing.adoc [hunk 1]
```diff
@@ -72,12 +72,11 @@ spec:
     matchLabels:
       pools.operator.machineconfiguration.openshift.io/worker: "" <2>
   kubeletConfig:
-    cpuCfsQuota: <3>
-      - "true"
+    cpuCfsQuota: false <3>
```
Comment (dfitzmau) [inline modules/nodes-cluster-overcommit-node-enforcing.adoc:75]: @xenolinux . Change LGTM and the preview looks good. I just want to confirm that dropping the quotation marks is correct?

"true" versus false.

@mburke5678 . Would you be OK to validate my peer review?

Comment (xenolinux) [reply modules/nodes-cluster-overcommit-node-enforcing.adoc:75]: @sunilcio Can you please help me with this question? https://github.com/openshift/openshift-docs/pull/63734#discussion_r1307417222

Comment (xenolinux) [reply modules/nodes-cluster-overcommit-node-enforcing.adoc:75]: > @xenolinux . Change LGTM and the preview looks good. I just want to confirm that dropping the quotation marks is correct?
> 
> "true" versus false.

Confirmed: It is correct

**Pull request merged**